app/processing/ignore-zero-value-splits.js

```diff
@@ -0,0 +1,30 @@
+const { AP } = require('../constants/ledgers')
+
+const ignoreZeroValueSplits = (paymentRequests) => {
+  if (!paymentRequests.length) {
+    return []
+  }
+
+  const paymentRequestsToRemove = []
+
+  for (const paymentRequest of paymentRequests) {
+    for (const otherPaymentRequest of paymentRequests) {
+      if (
+        paymentRequest !== otherPaymentRequest &&
```

Comment:
Could this be made more specific too? ie use the number here too?

Reply:
Refactored to a filter directly rather than looping to help with performance, let me know if you had something else in mind.

Reply:
made a further tweak to avoid doing `paymentRequest !== otherPaymentRequest` because I just didn't like doing it.

Checking value is non-zero should be sufficient given we then check `paymentRequest.value === -otherPaymentRequest.value`